plugins/woocommerce/includes/class-wc-countries.php

```diff
@@ -135,12 +135,7 @@ public function get_continent_code_for_country( $cc ) {
 	 * @return string|array Some countries have multiple. The code will be stripped of - and spaces and always be prefixed with +.
 	 */
 	public function get_country_calling_code( $cc ) {
-		$codes = wp_cache_get( 'calling-codes', 'countries' );
-
-		if ( ! $codes ) {
-			$codes = include WC()->plugin_path() . '/i18n/phone.php';
-			wp_cache_set( 'calling-codes', $codes, 'countries' );
-		}
```

Comment:
I think it probably makes sense to merge this (I fully understand and agree with the rationale) *...but...* with my back-compat hat on, is it possible there are folks out there who, rightly or wrongly, depend on caching as a means of customizing this information (which I don't think is filterable through normal means at present)?

Even if that might be the case, I think it's reasonable for us to say that 3PDs should not be doing this, but I wanted to pause and ask the question before proceeding.